.circleci/config.yml

```diff
@@ -981,6 +982,44 @@ jobs:
                   command: |
                       mvn clean deploy --activate-profiles gravitee-release --batch-mode -DskipTests -Dskip.validation=true --settings .gravitee.settings.xml --update-snapshots
 
+    run_postman_tests:
+        docker:
+            - image: cimg/openjdk:11.0-node
+              environment:
+                  - tcp.port: 8083
+            - image: mongo:4.4
+            - image: docker.elastic.co/elasticsearch/elasticsearch:7.10.1
+              environment:
+                  - transport.host: localhost #2
+                  - network.host: 127.0.0.1 #3
+                  - http.port: 9200 #4
+                  - cluster.name: es-cluster #5
+                  - discovery.type: single-node #6
+                  - xpack.security.enabled: false #7
+                  - ES_JAVA_OPTS: "-Xms256m -Xmx256m" #8
+        resource_class: small
+        steps:
+            - checkout
+            - attach_workspace:
+                  at: .
+            - run:
+                  name: Wait for Mongo to be up and running
+                  command: timeout 15s bash -c 'until nc -vz localhost 27017; do sleep 2; done'
+            - run:
+                  name: Install Newman
+                  command: sudo npm i -g newman
+            - run:
+                  name: Run API and Postman tests
+                  command: |
+                      # Start APIM in background
+                      GIO_MAX_MEM=1024m rest-api-docker-context/distribution/bin/gravitee &
```

Comment:
So we can remove [this](https://github.com/gravitee-io/gravitee-api-management/pull/1551/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R285) added in previous commit ?

Reply:
Done